C/services/north/north.cpp

```diff
@@ -402,7 +406,18 @@ void NorthService::start(string& coreAddress, unsigned short corePort)
 
 		m_storage->registerManagement(m_mgtClient);
 
-		// Fetch Confguration
+		// Setup the performance monitor
+		m_perfMonitor = new PerformanceMonitor(m_name, m_storage);
+
+		if (m_configAdvanced.itemExists("perfmon"))
```

Comment:
I believe consistency wise this should be in camelCase i.e `perfMon` as other advanced config items were with same naming convention.

Same applies to South side